Zend/zend_closures.c

```diff
@@ -107,10 +106,8 @@ static zend_bool zend_valid_closure_binding(
 
 	return 1;
 }
-/* }}} */
 
-/* {{{ proto mixed Closure::call(object to [, mixed parameter] [, mixed ...] )
-   Call closure, binding to a given object with its class as the scope */
+
```

Comment:
It might be better to remove the empty line.

Reply:
I tried my best to remove empty lines that were added but I didn't always manage to do so :( I'll have another look though

Reply:
I've just seen that there are way too many new lines added :/ So probably I'll have to retry as soon as we agree that protos/folding marks are ok to remove